README.md

```diff
@@ -129,51 +129,89 @@ the `encryption_*` and `ssl_*` options
-irods.client_init.write_pam_credentials_to_secrets_file
+
+By default, when an `.irodsA` file already exists, it will be overwritten. If however the
+`overwrite` parameter is set to `False`, an exception of type `irods.client_init.irodsA_already_exists`
+is raised to warn of any older `.irodsA` file that might otherwise have been overwritten.
+
+Equivalently to the above, we can issue the following command.
+
+```bash
+bash$ prc_write_irodsA.py native <<<"${MY_CURRENT_IRODS_PASSWORD}"
 ```
 
-Each takes a cleartext password and writes an appropriately processed version of it
-into an .irodsA (secrets) file in the login environment.
+The redirect may of course be left off, in which case the user is prompted for the iRODS password
+and echo of the keyboard input will be suppressed, in the style of `iinit`.  Regardless
+which technique is used, no password will be visible on the terminal during or after input.
+
+For the `pam_password` scheme, typically SSL/TLS must first be enabled to avoid sending data related
+to the password - or even sending the raw password itself - over a network connection in the clear.
 
-Examples:
-For the `native` authentication scheme, we can use the currently set iRODS password to create the .irodsA file directly:
+Thus for `pam_password` authentication to work well, we should first ensure, when setting up the
+client environment, to include within `irods_environment.json` the appropriate SSL/TLS connection
+parameters.  In a pinch, `iinit` can be used to verify this prerequisite is fulfilled,
+as its invocation would then create a valid .irodsA from merely prompting the user for their PAM password.
+
+Once again, this can also be done either using the free function directly:
 
 ```python
-import irods.client_init as iinit
-iinit.write_native_credentials_to_secrets_file(irods_password)
+irods.client_init.write_pam_irodsA_file(getpass.getpass('Enter current PAM password -> '))
 ```
 
-Note, in the `pam_password` case, this involves sending the cleartext password
-to the server (SSL must be enabled!) and then writing the scrambled token that
-is returned from the transaction.
+or from the Bash command shell:
 
-If an .irodsA file exists already, it will be overwritten by default; however, if these functions'
-overwrite parameter is set to `False`, an exception of type `irods.client_init.irodsA_already_exists`
-will be raised to indicate the older .irodsA file is present.
+```bash
+bash$ prc_write_irodsA.py pam_password <<<"${MY_CURRENT_PAM_PASSWORD}"
+```
 
-For the `pam_password` authentication scheme, we must first ensure an `irods_environment.json` file exists in the 
-client environment (necessary for establishing SSL/TLS connection parameters as well as obtaining a PAM token from the server after connecting)
-and then make the call to write .irodsA using the Bash commands:
+As a final note, in the "pam_password" scheme the default SSL requirement can be disabled (preferably for
+purposes of testing only!):
```

Comment:
I think we should include a stronger warning against doing this outside of testing. I leave the wording up to you, but maybe something like this?

```suggestion
As a final note, in the "pam_password" scheme the default SSL requirement can be disabled.
**Warning:** Disabling the SSL requirement may cause user passwords to be sent over the network
in the clear. This should only be done for purposes of testing. Here's how to do it:
```